taxi/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    def __str__(self) -> str:
+        return f"{self.name} ({self.country})"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=50, unique=True)
+
```

Comment:
are you sure you pushed all migrations? Because in the last migration I see you set default value, but in the model there isn't a default value
<img width="442" alt="image" src="https://github.com/user-attachments/assets/7605282b-a4db-4fb0-b7dd-0dcaec2151b8">

Reply:
Yeah, I forgot to make it in my terminal(